packages/ui-library/src/components/internal-components/form-caption/index.stories.ts

```diff
@@ -58,14 +58,6 @@ export default {
         category: 'Content / Setting',
       },
     },
-    arialabel: {
```

Comment:
Does this mean that this property needs to be removed from all other components or just the formCaption?